modules/serverless-rn-1-26-0.adoc

```diff
@@ -11,6 +11,7 @@
 [id="new-features-1.26_{context}"]
 == New features
 
+* {FunctionsProductName} is now GA.
```

Comment:
Outside the scope of this PR: I see we're using the term, "GA", widely. How might we ensure that its meaning is clear to all users?

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#release-notes

Reply:
This is a good point. I looked at other instances in our docs and the preferred representation is "General Availability (GA)", but the Serverless style seems to be just "GA" so I went with that here but feet the other way is better. 

I agree that we need to look at this to make it consistent across products.

Reply:
I suppose that for the Serverless the reason for shortened "GA" is that it is expanded in the top "Generally Available and Technology Preview features" section of the Release Notes.